jsAgent/links.js

```
@@ -85,17 +85,20 @@ bw.links={
             const link = links[i];
             let content_link = global_content_html.replace(new RegExp('{{link}}', 'g'), link.url );
             content_link = content_link.replace(new RegExp('{{title}}', 'g'), link.title );
+            content_link = content_link.replace(new RegExp('{{id}}', 'g'), link.id );
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)